bitnami/kafka/README.md

```diff
@@ -119,6 +121,8 @@ The following tables lists the configurable parameters of the Kafka chart and th
 | `auth.jaas.zookeeperUser`                         | Kafka Zookeeper user for SASL authentication                                                                                      | `nil`                                                   |
 | `auth.jaas.zookeeperPassword`                     | Kafka Zookeeper password for SASL authentication                                                                                  | `nil`                                                   |
 | `auth.jaas.existingSecret`                        | Name of the existing secret containing credentials for brokerUser, interBrokerUser and zookeeperUser                              | `nil`                                                   |
+| `auth.jaas.clientUsers`                           | List of Kafka client users to be created, separated by commas. This values will override `auth.jaas.clientUser`                   | `nil`                                                   |
+| `auth.jaas.clientPasswords`                       | List of passwords for `auth.jaas.clientUsers`. It is mandatory to provide the passwords when using `auth.jaas.clientUsers`        | `nil`                                                   |
```

Comment:
Remove `auth.jaas.brokerUser` and `auth.jaas.brokerPassword` from the table (those names were deprecated in favor of `auth.jaas.clientUser` but I forgot to update it)

Reply:
👍